bucket/shadowsocks-v2ray-plugin.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "1.3.1",
+    "description": "Yet another SIP003 plugin for shadowsocks, based on v2ray",
+    "homepage": "https://github.com/shadowsocks/v2ray-plugin",
+    "license": "MIT",
+    "depends": "shadowsocks",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/shadowsocks/v2ray-plugin/releases/download/v1.3.1/v2ray-plugin-windows-amd64-v1.3.1.tar.gz",
+            "hash": "sha1:8b7ebfbb98a26f5a9cb206a631bda17fcd418bd6",
+            "bin": [
+                [ "v2ray-plugin_windows_amd64.exe", "v2ray-plugin"]
+            ]
+        },
+        "32bit": {
+            "url": "https://github.com/shadowsocks/v2ray-plugin/releases/download/v1.3.1/v2ray-plugin-windows-386-v1.3.1.tar.gz",
+            "hash": "sha1:fcb25cae77b392178efb91c6fb2450fb3f9e7da3",
+            "bin": [
+                ["v2ray-plugin_windows_386.exe", "v2ray-plugin"]
+            ]
```

Completely nonsense name.

```suggestion
            "bin": [
                [
                    "v2ray-plugin_windows_386.exe",
                    "shadowsocks-v2ray-plugin"
                ]
            ]
```